src/wp-content/themes/twentyeleven/inc/widgets.php

```diff
@@ -156,6 +156,10 @@ public function widget( $args, $instance ) {
 	 * where any validation should be dealt with.
 	 *
 	 * @since Twenty Eleven 1.0
+	 *
+	 * @param array $new_instance New settings for this instance as input by the user.
+	 * @param array $old_instance Old settings for this instance.
+	 * @return array Updated settings to save.
```

Comment:
Good catch!

[Twenty Fourteen has a similar function](https://github.com/WordPress/wordpress-develop/blob/682f534498caf6f48d4985ad553d9baa5cfef19a/src/wp-content/themes/twentyfourteen/inc/widgets.php#L258-L270), and this _could_ be closer to that theme's docblock:
```suggestion
	 * @param array $new_instance New widget instance.
	 * @param array $old_instance Original widget instance.
	 * @return array Updated widget instance.
```

Reply:
The sentence 
  "Here is where any validation should be dealt with." 
could also be changed to something like 
  "Any validation should happen here."